docs/journalist/introduction.rst

```diff
@@ -6,5 +6,84 @@ Introduction
 What is Qubes OS?
 -----------------
 
+`Qubes OS`_ is an open source, security-focused
+operating system. It is very different than operating systems you may be
+familiar with already, because the operating system consists of multiple
+isolated virtual machines (called qubes), which allow you to separate more
+trusted components, files, or programs on your computer from less trusted
+components, files, or programs.
+
+Broadly speaking, this means that even if one of your qubes is
+exposed to malware, your other qubes can still be protected, which is not true
+of other operating systems.
+
+.. _`Qubes OS`: https://www.qubes-os.org
+
+How is using Qubes different from using virtual machines?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Virtual machines that run on your Mac, Windows, or Linux machine (such as those
+created using VirtualBox, Parallels, and so on) are a "guest" on your machine,
+but still require a "host" operating system on top of which to run. These types
+of virtual machines are not designed as security tools; if the host OS is
+compromised, there are no protections for the guest OS, and some features (such
+as networking) allow communications between guest and host that can compromise
+the security of both.
+
+In contrast, Qubes virtualization occurs at a lower level, under the `Xen
+hypervisor`_. This means that virtual machines (qubes) in a Qubes environment
+can run operating systems that are isolated from and independent of each
+other. In addition, qubes can be used to quarantine specific functions of your
+computer. For example, network access is provided via two or more
+qubes, and you can control which applications or files
+have access to a networked environment by connecting to or disconnecting from
+these qubes.
+
+Finally, Qubes is designed to make it more difficult for malware to remain on
+your machine. Each qube has read-only access to the root filesystem that
+provides the operating system for that qube, meaning that if a qube is infected
+with malware, it will be more difficult for that malware to persist across a
+reboot of that qube.
+
+For more about the security features of Qubes, see
+`the Qubes OS documentation`_.
+
+.. _`Xen hypervisor`: https://wiki.xen.org/wiki/Xen_Project_Software_Overview
+.. _`the Qubes OS documentation`: https://www.qubes-os.org/faq/#general--security
+
 What is SecureDrop Workstation?
```

Comment:
(This text pre-dates your PR, so just making a note here to discuss.)

We're using "SecureDrop Workstation" as a proper noun with no article in a couple of places. I am OK with this (SecureDrop 3000, here we come), but if we do it, we should standardize usage a bit, e.g., "the SecureDrop Workstation computer" for the physical hardware it's installed on.

Alternatively, we could use a definite article in most usage - "the SecureDrop Workstation". That is how I think we've been using it everywhere else so far (e.g., most uses in the README), but I may have missed uses like the above.

Reply:
I've been using *SecureDrop Workstation* as a proper noun in the admin docs, so this is consistent with those. It seems appropriate as it's an application. Precision > mild awkwardness imo.

Reply:
As per @zenmonkeykstop, removed uses of "The SecureDrop Workstation" in favour of "SecureDrop Workstation" and "the/your computer"

Reply:
:heavy_check_mark: